tools/update_info.py

```diff
@@ -10,44 +10,57 @@
 import itertools
 from collections import defaultdict
 
-with open('cookiecutter.json') as f:
+with open("cookiecutter.json") as f:
     data = json.load(f)
 
-db_resources = data['__prompts__']['db_resource'].items()
-web_frameworks = data['__prompts__']['project_backend'].items()
-deployment_hosts = data['__prompts__']['project_host'].items()
+db_resources = data["__prompts__"]["db_resource"].items()
+web_frameworks = [x for x in data["__prompts__"]["project_backend"].items() if x[0] == "fastapi"]
```

Comment:
Revert?

Reply:
yeah I should do that huh
```suggestion
web_frameworks = data["__prompts__"]["project_backend"].items()
```